### ISSUE TITLE
Fix validity of autogenerated destructuring namepaths

### ISSUE THREAD

**Issue by brettz9** [gajus/eslint-plugin-jsdoc#711]
Unfortunately, autogenerated destructuring comments like:

```js
  /**
   * @param root0
   * @param root0.0
   * @param root0.1
   */
```

...are invalid namepaths. It should instead produce:

```js
  /**
   * @param root0
   * @param root0."0"
   * @param root0."1"
   */
```
